<!--REQUEST_ID=b33caeef-07fc-429f-b9b2-f21a1d20f804-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.51130&attempt_id=aff629d7-ac31-4489-b5b4-781eab268fc1)

Attempted: x42-plugins

<details><summary>Partial log (click to expand)</summary><p>

```
  -o build/balance.so balance.c \
  -shared -Wl,-Bstatic -Wl,-Bdynamic  -lm
g++: error: unrecognized command line option '-msse'; did you mean '-fdse'?
g++: error: unrecognized command line option '-msse2'
g++: error: unrecognized command line option '-mfpmath=sse'
make[1]: *** [Makefile:192: build/balance.so] Error 1
make[1]: Leaving directory '/build/x42-plugins-20181103/balance.lv2'
make: *** [Makefile:34: balance.lv2] Error 2
builder for '/nix/store/0xs62sqdfinkkgyfp4wdxs1p0vphag2n-x42-plugins-20181103.drv' failed with exit code 2
error: build of '/nix/store/0xs62sqdfinkkgyfp4wdxs1p0vphag2n-x42-plugins-20181103.drv' failed
```
</p></details>

--- a/pkgs/applications/audio/x42-plugins/default.nix
+++ b/pkgs/applications/audio/x42-plugins/default.nix
@@ -3,12 +3,12 @@
 , libGLU, lv2, gtk2, cairo, pango, fftwFloat, zita-convolver }:
 
 stdenv.mkDerivation rec {
-  version = "20180812";
+  version = "20181103";
   name = "x42-plugins-${version}";
 
   src = fetchurl {
     url = "https://gareus.org/misc/x42-plugins/${name}.tar.xz";
-    sha256 = "0gzwzxpa2k2w9c6j3pspwi9slfyd57wb192d6yqcg92pfmnxy9dz";
+    sha256 = "085d6qjj7nl22f0xamqdrnfxwi8zrfwgkwm1svm73bjkdv270438";
   };
 
   nativeBuildInputs = [ pkgconfig ];
@@ -34,6 +34,6 @@ stdenv.mkDerivation rec {
       homepage = https://github.com/x42/x42-plugins;
       maintainers = with maintainers; [ magnetophon ];
       license = licenses.gpl2;
-      platforms = platforms.linux;
+      platforms = [ "i686-linux" "x86_64-linux" ];
     };
 }

